src/main/kotlin/net/maizegenetics/phgv2/cli/BuildRefVcf.kt

```diff
@@ -1,13 +1,37 @@
 package net.maizegenetics.phgv2.cli
 
+import biokotlin.genome.fastaToNucSeq
+import biokotlin.seq.NucSeq
+import biokotlin.util.bufferedReader
 import com.github.ajalt.clikt.core.CliktCommand
 import com.github.ajalt.clikt.parameters.options.default
 import com.github.ajalt.clikt.parameters.options.option
 import com.github.ajalt.clikt.parameters.options.validate
+import com.google.common.collect.HashMultimap
+import com.google.common.collect.Multimap
+import com.google.common.collect.Range
+import htsjdk.variant.variantcontext.VariantContext
+import htsjdk.variant.vcf.VCFAltHeaderLine
+import htsjdk.variant.vcf.VCFHeaderLine
+import htsjdk.variant.vcf.VCFHeaderVersion
+import net.maizegenetics.phgv2.utils.*
+import org.apache.logging.log4j.LogManager
+import java.io.File
+import java.util.*
+import java.util.logging.Logger
 
 
 class BuildRefVcf : CliktCommand() {
 
+    private val myLogger = Logger.getLogger("net.maizegenetics.phgv2.cli.BuildRefVcf")
+
+    var myRefSequence: Map<String, NucSeq>? = null
+
+    // refurl is not required.  If present, it will result in a ##reference header
+    // in the hvcf file.
+    val refurl by option(help = "URL where the reference FASTA file can be downloaded")
+        .default("")
+
     val bed by option(help = "BED file")
         .default("")
```

Comment:
I know you probably copied this from somewhere else. But I don't see the reason for have an empty default and then checking that it exists.  Can't this just be required()?  Same for all these parameters

Reply:
@tcasstevens The refurl is not required.  I default it to "" so the value can be checked later.  If refurl != "", then a "##reference=..." header is added to the hvcf file.

Reply:
@tcasstevens I mentioned this in another PR, but I tried to leave off the required() portion but it was not working correctly without the default("").  I am not sure why this is, but it did matter.